### PR TITLE
update CHANGES (master)

### PR DESCRIPTION
**Pull request by ThomasWaldmann**

### CODE REVIEW

File: docs/changes.rst [hunk 1]
```diff
@@ -155,13 +155,19 @@ Compatibility notes:
 - Repositories in the "authenticated" mode are now treated as the unencrypted
   repositories they are.
 
+- The client-side repository cache now holds unencrypted data for better speed.
```
Comment (enkore) [inline docs/changes.rst]: Perhaps "temporary repository cache" to make it more clear that this isn't about _the_ cache.

File: docs/changes.rst [hunk 2]
```diff
@@ -155,13 +155,19 @@ Compatibility notes:
...
 New features:
 
-- integrity checking for important files used by borg:
+- reimplemented the RepositoryCache, size-limited caching of decrypted repo
+  contents, integrity checked via xxh64.
+- integrity checking using xxh64 for important files used by borg:
 
   - repository: index and hints files
   - cache: chunks and files caches, archive.chunks.d
+- compact chunks.archive.d
```
Comment (enkore) [inline docs/changes.rst]: Maybe "Reduced space usage of chunks.archive.d. Existing caches are migrated during a cache sync".

File: docs/changes.rst [hunk 3]
```diff
@@ -180,18 +186,26 @@ Fixes:
...
-- serve: fix incorrect type of exception_short for Errors, #2513
 - fix remote logging and progress, #2241
 
   - implement --debug-topic for remote servers
   - remote: restore "Remote:" prefix (as used in 1.0.x)
   - rpc negotiate: enable v3 log protocol only for supported clients
   - fix --progress and logging in general for remote
+- fix parse_version, add tests, #2556
+- repository: truncate segments (and also some other files) before unlinking, #2557
+- algorithms.checksums: work around GCC 4.4 bug by disabling CLMUL, #2611
```
Comment (enkore) [inline docs/changes.rst]: the issue wasn't included in a release

File: docs/changes.rst [hunk 4]
```diff
@@ -240,6 +273,12 @@ Other changes:
   - testsuite.archiver: normalise pytest.raises vs. assert_raises
   - add test for preserved intermediate folder permissions, #2477
   - key: add round-trip test
+  - remove attic dependency of the tests, #2505
+  - enable remote tests on cygwin
+  - hashindex: Cython defines PY_SSIZE_T_CLEAN, #2634
```
Comment (enkore) [inline docs/changes.rst]: this was also not included in a release

File: docs/changes.rst (hunk 2 above)
Comment (enkore) [inline docs/changes.rst]: I think in the line above this one, chunks.archive.d is swapped around

Comment (ThomasWaldmann) [inline docs/changes.rst]: ok

File: docs/changes.rst (hunk 1 above)
Comment (ThomasWaldmann) [inline docs/changes.rst]: ok

File: docs/changes.rst (hunk 3 above)
Comment (ThomasWaldmann) [inline docs/changes.rst]: ok, removing 2611

File: docs/changes.rst (hunk 4 above)
Comment (ThomasWaldmann) [inline docs/changes.rst]: removing 2634

**Comment by enkore** [comment]
In the future it might be easier / quicker to edit larger changelog updates somewhere else (e.g. on the wiki) and merge it when done; PRs aren't well-suited to collaborative editing.

Incidentally, I gathered a first summary of changes for 1.0->1.1: https://github.com/borgbackup/borg/wiki/Borg-1.1#draft-release-announcement--change-summary

File: docs/changes.rst [hunk 5]
```diff
@@ -233,8 +232,11 @@ Other changes:
 - increase DEFAULT_SEGMENTS_PER_DIR to 1000
 - chunker: fix invalid use of types (function only used by tests)
 - chunker: don't do uint32_t >> 32
-- FUSE: add instrumentation / debug logging
-- FUSE: don't keep all Archive() instances around
+- fuse:
```
Comment (ThomasWaldmann) [inline docs/changes.rst]: "FUSE", see fixes section.

**Pull request merged**